CHANGELOG.md

```diff
@@ -24,7 +24,13 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.16.1...master
+[Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.17.0...master
+
+## [3.17.0] - 2022-04-13
```

Comment:
I don't think we released that. 

Our approach is to mark release in CANGELOG after it's released. Then it's certain what is inside it and when it was released.

You should add the `### Added` just to the `## [Unreleased]`